commit 9b2576e5deca655b4a5e65faf6eddb2e9b3ea979
Merge: 05932e4 f08bb8b
Author: ErikZalm <erik@vdzalm.eu>
Date:   Sun Jan 19 23:44:36 2014 -0800

    Merge pull request #725 from whosawhatsis/Marlin_v1
    
    Fix bugs in Extruder Runout Prevention, including DAMAGING HEAD CRASH

diff --git a/Marlin/Marlin_main.cpp b/Marlin/Marlin_main.cpp
index 26bc66f..920aed0 100644
--- a/Marlin/Marlin_main.cpp
+++ b/Marlin/Marlin_main.cpp
@@ -3298,8 +3298,8 @@ void manage_inactivity()
      enable_e0();
      float oldepos=current_position[E_AXIS];
      float oldedes=destination[E_AXIS];
-     plan_buffer_line(current_position[X_AXIS], current_position[Y_AXIS], current_position[Z_AXIS],
-                      current_position[E_AXIS]+EXTRUDER_RUNOUT_EXTRUDE*EXTRUDER_RUNOUT_ESTEPS/axis_steps_per_unit[E_AXIS],
+     plan_buffer_line(destination[X_AXIS], destination[Y_AXIS], destination[Z_AXIS], 
+                      destination[E_AXIS]+EXTRUDER_RUNOUT_EXTRUDE*EXTRUDER_RUNOUT_ESTEPS/axis_steps_per_unit[E_AXIS], 
                       EXTRUDER_RUNOUT_SPEED/60.*EXTRUDER_RUNOUT_ESTEPS/axis_steps_per_unit[E_AXIS], active_extruder);
      current_position[E_AXIS]=oldepos;
      destination[E_AXIS]=oldedes;

